dev_guide/pod_autoscaling.adoc

```diff
@@ -215,6 +230,11 @@ spec:
 $ oc create -f hpa.yaml
 ----
 
+[NOTE]
+====
+Verify that your application's memory usage does not go down, which is more complicated than it seems.
```

Comment:
@DirectXMan12 If we want to add a note for complexities we should include what is required? why it is required. 
```
Verify that your application's memory usage does not go down (go down from which value? the one we set in example?)
, which is more complicated than it seems. (why? what goes wrong if we don't do that)
```

Reply:
@DirectXMan12 PTAL

Reply:
Probably something like

```
In order for memory-based autoscaling to work, memory usage must increase *and* decrease proportionally to replica count.  More specifically, an increase in replica count *must* lead to an overall decrease in memory (working set) usage per pod, on average.  Similarly, a decrease in replica count *must* lead to an overall increase in per-pod memory usage, on average.

It's important to ensure that your application meets these requirements before trying to use memory-base d autoscaling.  Use the OpenShift dashboard (and `oc top` in 3.10+) to verify the memory behavior of your application.